assets/component-slideshow.css

```diff
@@ -3,24 +3,6 @@ slideshow-component {
   display: block;
 }
 
-.spaced-section--full-width:first-child slideshow-component.page-width {
-  margin-top: 5rem;
-}
-
-.spaced-section--full-width:last-child slideshow-component.page-width {
-  margin-bottom: 5rem;
-}
-
-@media screen and (min-width: 750px) {
-  .spaced-section--full-width:first-child slideshow-component.page-width {
-    margin-top: calc(5rem + var(--page-width-margin));
-  }
-
-  .spaced-section--full-width:last-child slideshow-component.page-width {
-    margin-bottom: calc(5rem + var(--page-width-margin));
-  }
-}
-
```

Comment:
Let's pair on this a bit tomorrow.

The featured collection is only getting it incidentally from the `.grid` class because the `ul` itself has a bottom margin.  I would actually argue that it should be removed from that because it's creating a gap that shouldn't really be there.  If the merchant has set the vertical spacing between sections to 0 and have it and the next section after it with the same colour, there would be an unwanted gap.

Reply:
Examples:

Current (with `2rem` on `.grid`):

<img width="1178" alt="Screen Shot 2022-01-07 at 4 41 20 PM" src="https://user-images.githubusercontent.com/60230011/148611143-8b16e90f-78d1-4fb0-88db-20909a9a4b2d.png">

Expected:

<img width="1179" alt="Screen Shot 2022-01-07 at 4 41 39 PM" src="https://user-images.githubusercontent.com/60230011/148611157-e716422c-c2d5-4415-9673-7a1145d339d7.png">

Reply:
I got rid of that incidental bottom margin when the grid is the `:last-child` to prevent this sort of thing.  If the merchant has set no vertical gap, then there should be no vertical gap.

Reply:
Ok let's do it but Monday instead 😜